src/kernel/inwards.jl

```diff
@@ -4,13 +4,10 @@
     InwardsPopulationDispersal{R}(; neighborhood)
     InwardsPopulationDispersal{R,W}(; neighborhood)
 
-Disperses to the current cells from the populations of the
-surrounding cells, using a dispersal kernel deterministically.
+Implements deterministic dispersal from populations in neighboring cells to the current cell. 
 
-This will make sense ecologically where cell populations are large, 
-otherwise a randomised kernel may be more suitable.
+The result should be identical to those obtained substituting `InwardsDispersal` for [`OutwardsDispersal`](@ref) but will perform better when populations are spread across the grid.
```

Comment:
We should keep the line length max 92 chars so you can still read it in the terminal/code editor

Reply:
I have now done this through out but for bullet points I had to keep them on the same line.  I thought two spaces at the end of the line would connect the next line as per [this](https://stackoverflow.com/questions/33673639/multi-line-bullet-list-in-markdown), but no luck.

Reply:
You don't need a line-break in the output text, so don't put the two spaces at the end. Just move to the next line and indent 4 spaces. But I can fix that in my other docs PR if your'e stuck on it.